modules/AOW_Actions/actions/actionCreateRecord.php

```diff
@@ -282,19 +281,21 @@ public function set_record(SugarBean $record, SugarBean $bean, $params = array()
                                 break;
                         }
                         break;
-                    case 'Round_Robin':
```

Comment:
@gymad - we happy with the deference of 'Case' versus 'case'? :)

Reply:
You right @samus-aran, I didn't realized this one, fortunatelly it does not breaks anything cause both are a valid part of the php lang. Also I think we will having much more issue about this handreds line long messy switch..case..else..if labyrinth here.

Reply:
I fixed this in a couple of new commits, also fixing other tiny objections from Codacy.